### PR TITLE
Cdn for images api

### PR DESCRIPTION
**Pull request by emmanue**
Images api caching in the dev env

### CODE REVIEW

File: cdn-images-api/main.tf [hunk 1]
```diff
@@ -0,0 +1,55 @@
+resource "aws_cloudfront_distribution" "images_cdn" {
+  optimizer {
+    origin {
+      domain_name = "dev-k8s.treetracker.org"
```
Comment (TANguyen1893) [inline cdn-images-api/main.tf:4]: Since this is for dev, will you also make a PR in the future for test and prod?

**Review by TANguyen1893 — APPROVED** [review]

File: cdn-images-api/main.tf (hunk 1 above)
Comment (emmanue) [reply cdn-images-api/main.tf:4]: Yes I will. First I will deploy in the dev env to check that everything works well, after that I will do same in the test and pro env.

**Comment by dadiorchen** [comment]
@emmanue this is good for me!

**Comment by dadiorchen** [comment]
@emmanue how do we set the expire time in the cdn?

**Comment by emmanue** [comment]
@dadiorchen it will expire after 24 hours, unless you want it to expire earlier than that.

**Pull request merged**